index.bs

```diff
@@ -1167,6 +1167,17 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
             when the [=public key credential source=] was [=registration|registered=].
             Storing this in combination with the above [$credential record/attestationObject$] [=struct/item=]
             enables the [=[RP]=] to re-verify the [=attestation signature=] at a later time.
+
+        :   <dfn>displayName</dfn>
```

Comment:
Shouldn't the initial value of this field always be equal to [user.displayName](https://w3c.github.io/webauthn/#dictdef-publickeycredentialuserentityjson)? And shouldn't credProps.userDisplayName (always) return the original user.displayName value?

Reply:
I see. In this case `user.displayName` sounds redundant, because I can distinguish my accounts by `user.name`, but this is a different story. Thanks for clarification.

Reply:
Maybe a different name makes it clearer on first read? E.g. credentialStoreNickname or something in that vein?

Reply:
Additional feedback during today's WG meeting was that we should return a single value to the RP; I've merged both properties and renamed it to `authenticatorDisplayName`.